src/api/index.py

```diff
@@ -20,18 +20,72 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
 
+from enum import Enum
 from typing import Optional
 
 from fastapi import FastAPI, HTTPException
 
 from src.classes.analyze_video_response import AnalyzeVideoResponse
-from src.services.image_analysis_service import run_image_analysis
-from src.services.video_service import download_video
+from src.classes.video_data import VideoData
+from src.classes.yolo_model import YoloModel
+from src.services.image_analysis_service import AnalyzerService
+from src.services.prep_service import DataPreparationService
+from src.utils import default_variables as dv
 from src.utils.default_variables import target_item, video_url
 
 app = FastAPI()
 
 
+class ModelChoices(str, Enum):
+    yolov5 = "yolov5"
+    yolov8 = "yolov8"
+
+
+def get_model_attributes(model_selection: str):
+    """
+    Function to determine the type of model family and
+    model version to use
+
+    Parameters
+    -------------
+    model_selection : str
+        Type of model to use. Options: [`yolov5`, `yolov8`]
+
+    Returns
+    -----------
+    model_config : dict
+        Dictionary containing the set of configuration to
+        use for the specified model.
+    """
+    if model_selection == "yolov5":
```

Comment:
We can update this code to use the new enum value. This should work since you used `str` when defining the class. We can update line 62 as well for v8.
```suggestion
    if model_selection == ModelChoices.yolov5:
```